dev/com.ibm.ws.jpa_ol_fat/bnd.bnd

```diff
@@ -0,0 +1,34 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+
+src: \
+	fat/src,\
+	test-applications/ConcurrentEnhancementVerification/src,\
+	test-applications/helpers/DatabaseManagement/src,\
+	test-framework/src
+
+fat.project: true
+tested.features: el-3.0, jpa-3.0, servlet-5.0, cdi-3.0, beanValidation-3.0, ejblite-4.0, jdbc-4.0, beanvalidation-1.1, jpa-2.0
```

Comment:
I asked outside of the issue, but it seems weird to have to have cdi, el and beanvalidation features be enabled by the repeats when you do not have them in your server.xml file.  That seems like a test framework bug.